inseq/attr/feat/attention_attribution.py

```diff
@@ -114,53 +120,39 @@ def attribute_step(
             step_scores={},
         )
 
-    @classmethod
-    def load(
-        cls,
-        method_name: str,
-        attribution_model=None,
-        model_name_or_path: Union[ModelIdentifier, None] = None,
-        **kwargs,
-    ) -> "FeatureAttribution":
-        from inseq import AttributionModel
-
-        if model_name_or_path is None == attribution_model is None:  # noqa
-            raise RuntimeError(
-                "Only one among an initialized model and a model identifier "
-                "must be defined when loading the attribution method."
-            )
-        if model_name_or_path:
-            attribution_model = AttributionModel.load(model_name_or_path)
-            model_name_or_path = None
-
-        if not attribution_model.model.config.output_attentions:
```

Comment:
Would it not make sense to leave this behavior in the class? Just in case an already instantiate model is passed in that did not enable `output_attentions` in its config?

Reply:
Ah ok, I just saw that this is now included in the arguments for the forward pass.